tasks/src/main/java/ru/sberbank/school/task09/Route.java

```diff
@@ -0,0 +1,44 @@
+package ru.sberbank.school.task09;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Route<C extends City> {
+    private String routeName;
+    private List<C> cities = new LinkedList<>();
+
+    public Route() {
+
+    }
+
+    public Route(String routeName, List<C> cities) {
+        this.routeName = routeName;
+        this.cities = cities;
+    }
+
+    public String getRouteName() {
+        return routeName;
+    }
+
+    public void setRouteName(String routeName) {
+        this.routeName = routeName;
+    }
+
+    public List<C> getCities() {
+        return cities;
+    }
+
+    public void setCities(List<C> cities) {
+        this.cities = cities;
+    }
+
+    @Override
+    public String toString() {
+        return "Route: { " +
```

Comment:
&#39;+&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a703056006>*